velox/exec/HashAggregation.cpp

```diff
@@ -117,7 +123,20 @@ bool HashAggregation::abandonPartialAggregationEarly(int64_t numOutput) const {
       100 * numOutput / numInputRows_ >= abandonPartialAggregationMinPct_;
 }
 
+void HashAggregation::processMergedVectors(bool noMoreInput) {
+  auto mergedVector = mergingVectorInput_->getVector(noMoreInput);
+  while (nullptr != mergedVector) {
+    addInputInternal(mergedVector);
+    mergedVector = mergingVectorInput_->getVector(noMoreInput);
+  }
+}
+
 void HashAggregation::addInput(RowVectorPtr input) {
+  mergingVectorInput_->addVector(input);
+  processMergedVectors(noMoreInput_);
```

Comment:
Here, noMoreInput_ must be false. It would be cleaner to us 'false' constant.

Reply:
Done.